okhttp/src/test/java/okhttp3/InterceptorTest.java

```diff
@@ -679,7 +680,8 @@ private void interceptorThrowsRuntimeExceptionAsynchronous(boolean network) thro
       return chain.proceed(chain.request());
     };
 
-    ServerSocket serverSocket = new ServerSocket(0, 1);
+    InetAddress localhost = InetAddress.getLoopbackAddress();
```

Comment:
We should do this for MockWebServer also, it's used in the start method

Reply:
MockWebServer calls `InetAddress.getByName("localhost")`, are you suggesting to change that for `InetAddress. getLoopbackAddress()`? Or do you mean something else?

Reply:
Yes, what do you think?  I think in practice it can be different, but shouldn't.

https://tools.ietf.org/html/rfc6761

> 6.3.  Domain Name Reservation Considerations for "localhost."
> 
>    The domain "localhost." and any names falling within ".localhost."
>    are special in the following ways:
> 
>    1.  Users are free to use localhost names as they would any other
>        domain names.  Users may assume that IPv4 and IPv6 address
>        queries for localhost names will always resolve to the respective
>        IP loopback address.
> 
>    2.  Application software MAY recognize localhost names as special, or
>        MAY pass them to name resolution APIs as they would for other
>        domain names.
> 
>    3.  Name resolution APIs and libraries SHOULD recognize localhost
>        names as special and SHOULD always return the IP loopback address
>        for address queries and negative responses for all other query
>        types.  Name resolution APIs SHOULD NOT send queries for
>        localhost names to their configured caching DNS server(s).

Reply:
I want to keep it as is because we've had problems with other addresses.

Reply:
I've not personally encountered issues with what `MockWebServer` was doing, so I'd leave it as is too.